sunpy/net/base_client.py

```diff
@@ -203,7 +203,9 @@ def _print_client(client, html=False):
     if html:
         lines = [f"<p>{line}</p>" for line in lines]
     lines.extend(t.pformat_all(show_dtype=False, max_width=width, align="<", html=html))
-    return '\n'.join(lines)
+    if visible_entries is None:
+        return '\n'.join(lines)
+    return '\n'.join(lines[0:visible_entries])
```

Comment:
```suggestion
    return '\n'.join(lines[slice(0:visible_entries)])
```

Reply:
Hi, @nabobalis ```lines[slice(0:visible_entries)``` was giving an error so I used ```lines[slice(visible_entries)]``` instead.

Reply:
Yeah, I made a big mistake in my suggestion.